content/docs/user-guide/merge-conflicts.md

```diff
@@ -0,0 +1,198 @@
+# Merge Conflicts
+
+Sometimes multiple members of a team might want to edit the same file/directory
+that is tracked by DVC. And when the time comes to merge their changes together
+one will run into a merge conflict.
+
+## dvc.yaml
+
+This one is no different from your regular merge conflicts in code.
+
+```
+stages:
+  prepare:
+    cmd: python src/prepare.py data/data.xml
+    deps:
+<<<<<<< HEAD
+    - data/big.xml
+=======
+    - data/small.xml
+>>>>>>> branch
+    - src/prepare.py
+    params:
+    - prepare.seed
+    - prepare.split
+    outs:
+    - data/prepared
+```
+
+See https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging for
+more details.
+
+## dvc.lock
+
+You can safely delete this file and then just run `dvc repro` after the merge is
+done.
+
+```
+prepare:
+  cmd: python src/prepare.py data/data.xml
+  deps:
+<<<<<<< HEAD
+  - path: data/big.xml
+=======
+  - path: data/small.xml
+>>>>>>> branch
+    md5: 285af85d794bb57e5d09ace7209f3519
+  - path: src/prepare.py
+<<<<<<< HEAD
+    md5: a304afb96060aad90176268345e10355
+=======
+    md5: 0d2602f440d13372435d91d3bfa17652
+>>>>>>> branch
+  params:
+    params.yaml:
+      prepare.seed: 20170428
+      prepare.split: 0.2
+  outs:
+  - path: data/prepared
+    md5: 20b786b6e6f80e2b3fcf17827ad18597.dir
+```
+
+## \*.dvc
+
+There are three three types of such files that differ by the command that has
+created them.
+
+### dvc import
+
+Simply remove the hashes
+
+```
+<<<<<<< HEAD
+md5: 263395583f35403c8e0b1b94b30bea32
+=======
+md5: 520d2602f440d13372435d91d3bfa176
+>>>>>>> branch
+frozen: true
+deps:
+- path: get-started/data.xml
+  repo:
+    url: https://github.com/iterative/dataset-registry
+<<<<<<< HEAD
+    rev_lock: f31f5c4cdae787b4bdeb97a717687d44667d9e62
+=======
+    rev_lock: 06be1104741f8a7c65449322a1fcc8c5f1070a1e
+>>>>>>> branch
+outs:
+<<<<<<< HEAD
+- md5: a304afb96060aad90176268345e10355
+=======
+- md5: 35dd1fda9cfb4b645ae431f4621fa324
+>>>>>>>
+  path: data.xml
+```
+
+so you get
+
+```
+frozen: true
+deps:
+- path: get-started/data.xml
+  repo:
+    url: https://github.com/iterative/dataset-registry
+outs:
+- path: data.xml
+```
+
+and then run `dvc repro`.
+
+### dvc import-url
+
+Simply remove the hashes
```

Comment:
Same as `dvc import` so maybe just combine the H3 headers and merge these sections?

Reply:
The yamls are different, so better to keep them separate.

Reply:
Exact YAML samples belong to those cmd refs I think. In terms of the merging, which is what this doc is about, seems to be the same so I think we could summarize the doc substantially by merging them 🙂

Reply:
Those yamls are not the same and illustrate conflicts in different places, let's leave them be.

Reply:
Seems like i'm taking over this one so will combine them. Thanks!

Reply:
@jorgeorpinel Your call, but I wouldn't combine them.

Reply:
Just noticed your comments here Ruslan, sorry. Hmmm WDYT @shcheklein ? I think if the process is the same for import and import-url then a single section is enough and prevents us from having to maintain 2 YAML examples. On the other hand there's no super need to optimize the last part of the doc so no very strong opinion, actually.